lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -45,13 +45,21 @@ def install!
                   create_build_phase_to_move_static_framework_archive
                 end
               end
-              create_prefix_header
+              unless skip_pch?
+                create_prefix_header
+              end
               create_dummy_source
             end
           end
 
           private
 
+          # @return [Boolean] Whether the target should build a pch file.
+          #
+          def skip_pch?
+            target.file_accessors.any? { |file_accessor| file_accessor.prefix_header == ':skip' }
```

Comment:
any reason to query each file accessor? I believe that method asks the spec itself which i think you have this information here.

I am only saying this because file accessors tend to be slow generally...

Reply:
You're right. Changed. Thanks!